delivery/overview.mdx

```diff
@@ -134,7 +134,7 @@ File sink currently supports only append-only mode, so please change the query t
 ## Batching strategy for file sink
 
 <Note>
-Batching strategy for file sink is newly introduced in version 2.1 and is not supported in versions prior to 2.1. If you want to use it, please upgrade to version 2.1.
+Batching strategy for file sink is only available since version 2.1. Please upgrade to use it.
```

Comment:
What about just 
```suggestion
Batching strategy for file sink is available since version 2.1.
```

"Please upgrade to use it" seems self-evident